armi/operators/settingsValidation.py

```diff
@@ -107,9 +105,7 @@ def resolve(self):
                         )
                         if make_correction:
                             self.correction()
-                            self._corrected = True
-                        else:
-                            self._passed = True
+                        self._passed = True
                     except RunLogPromptCancel as ki:
                         raise KeyboardInterrupt from ki
                 else:
```

Comment:
The `_corrected` attribute was not being used for anything and therefore it was removed. Related to debugging on the setting validators.

Reply:
I can add it back in. Nothing was using it. You're right that it's not a good place in this PR (same as above). I big thing here is that self._passed should be set to True regardless.